jstreegrid.js

```diff
@@ -1090,7 +1187,7 @@
 						width = tmpWidth || (width - paddingleft);
 					}
 
-					last = findDataCell(uniq, lid, i);
+          last = findDataCell(uniq, lid, i, column);
```

Comment:
Comes back to the `scope` question. What does adding `column` to `findDataCell()` do? Is it basically to allow it to search in detached? In other words, without `scope`, it would search in the rendered DOM, so now it searches in a detached node set?